practice6/src/main/java/com/academiaDigital/softwareArchitecture/Drivers/CsvDriver.java

```diff
@@ -0,0 +1,17 @@
+package com.academiaDigital.softwareArchitecture.Drivers;
+
+import com.academiaDigital.softwareArchitecture.singleton.Logger;
+
+public class CsvDriver implements IDriver {
+    Logger logger = Logger.getInstance();
+    @Override
+    public void read(String filename) {
+        logger.info();

```

Comment:
"normal" behavior of a logger would be to send a string to be logged: `logger.info("reading information")` instead of sending just `info()` without the message to be populated in the log.

Reply:
You are right, it's better if send a message.